packages/airnode-utilities/src/types.ts

```diff
@@ -0,0 +1,32 @@
+// ===========================================
+// Logging
+// ===========================================
+export type LogLevel = 'DEBUG' | 'INFO' | 'WARN' | 'ERROR';
+
+export type LogFormat = 'json' | 'plain';
+
+export interface LogMetadata {
+  readonly coordinatorId?: string;
```

Comment:
I don't know if we should use `readonly` for types on a utilities library... There might be valid cases where we want edit the properties. I know @Siegrift doesn't like these 😬 

I think a better option might be to use `Readonly<T>` where these types are used in the node/other packages.

Reply:
But this PR is already pretty big so I wouldn't do this as part of this branch (if you guys agree).